Some cleanup to the new text (#4446)
EnterpriseDB/docs: product_docs/docs/pgd/5/architectures.mdx

## Adding flexibility to the standard architectures

> You can deploy the single-location architecture in as many locations as you want. Doing so provides the data resiliency needed and proximity to applications and to the users maintaining the data. While EDB Postgres Distributed has a variety of conflict-handling approaches available, in general take care to minimize the number of expected collisions if allowing write activity from geographically disparate locations.

Reviewer: ```suggestion
To provide the data resiliency needed and proximity to applications and to the users maintaining the data, the single-location architecture can be deployed in as many locations as you want. While EDB Postgres Distributed has a variety of conflict-handling approaches available, do take care to minimize the number of expected collisions if allowing write activity from geographically disparate locations.
```
